### PR TITLE
[installer] improve ws-manager config

### PR DESCRIPTION
**Pull request by sagor999**
## Description

Same other PRs related to #10418, improves installer so that we don't need to overwrite configmap for SaaS

## Related Issue(s)

Related https://github.com/gitpod-io/gitpod/issues/10418

## Release Notes

```release-note
none
```

## Werft options:

- [ ] /werft with-preview

### CODE REVIEW

**Comment by roboquat** [comment]
@sagor999: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by sagor999** [comment]
/hold
until https://github.com/gitpod-io/gitpod/pull/11164 is merged (for fixed storage config)

File: install/installer/pkg/config/v1/experimental/experimental.go [hunk 1]
```diff
@@ -43,8 +44,13 @@ type PodConfig struct {
 }
 
 type WorkspaceConfig struct {
-	Tracing *Tracing `json:"tracing,omitempty"`
-	Stage   string   `json:"stage"`
+	Tracing                  *Tracing `json:"tracing,omitempty"`
+	Stage                    string   `json:"stage"`
+	SchedulerName            string   `json:"schedulerName"`
```
Comment (aledbf) [inline install/installer/pkg/config/v1/experimental/experimental.go:49]: this should be optional right?

Comment (sagor999) [reply install/installer/pkg/config/v1/experimental/experimental.go]: fixed!

**Review by aledbf — APPROVED** [review]

**Review by Pothulapati — APPROVED** [review]
LGTM :shipit:

**Pull request merged**